docs/pages/access-controls/guides/headless-login.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: Headless WebAuthn (Preview)
+description: Headless WebAuthn
+---
+
+<Notice type="warning">
+  Headless WebAuthn is currently in Preview mode.
+</Notice>
+
+Headless WebAuthn provides a secure way to authenticate with WebAuthn from a machine without access to a WebAuthn device. This enables use cases which are usually not with WebAuthn
```

Comment:
The last sentence here doesn't read quite right.

_which are usually not with WebAuthn features_